content/admin/org_settings/API-credentials.md

```diff
@@ -28,4 +28,4 @@ For more information, read [About the Contrast API](tools-about.html#api-about).
 
 To see more examples of API requests, go to the [Contrast API documentation](https://api.contrastsecurity.com/). 
 
-For help with connection issues after rotating service keys, read to the [troubleshooting article](troubleshooting-auth.html#keys). 
+For help with connection issues after rotating service keys, read to the [troubleshooting article](troubleshooting-auth.html#keys) <!-- replace -->. 
```

Comment:
@marklacasse I've marked the hyperlinks that need to be updated with new ones from the KB. Can you provide them here?

Reply:
For general Contrast UI issues I would link them here:
https://support.contrastsecurity.com/hc/en-us/sections/360000085563-Contrast-UI
This is under Troubleshooting --> Contrast UI
It does not look like I have a way to group specific to the install, looks like we are limited in the amount of sub-sections we can go. So this will have to do for now.

Reply:
I would remove the 
"For help with connection issues after rotating service keys, read to the [troubleshooting article](troubleshooting-auth.html#keys). " It points to a very old doc on crawler that we are not bringing back to life.